src/server/status/server_status.js

```diff
@@ -2,15 +2,24 @@ import _ from 'lodash';
 
 import states from './states';
 import Status from './status';
+import PluginStatus from './plugin_status';
 
 module.exports = class ServerStatus {
   constructor(server) {
     this.server = server;
     this._created = {};
   }
 
-  create(plugin) {
-    return (this._created[plugin.id] = new Status(plugin, this.server));
+  create(id) {
```

Comment:
My original instinct was to just list the ids for each status and let those draw their own distinction, if that's enough for you then it's enough for me!

![image](https://cloud.githubusercontent.com/assets/1329312/16067077/ca6dc552-326d-11e6-89b9-065e558109b3.png)